pkgs/by-name/an/ankama-launcher/package.nix

```
@@ -0,0 +1,35 @@
+{
+  appimageTools,
+  fetchurl,
+  lib,
+  writeScriptBin,
+}:
+let
+  pname = "ankama-launcher";
+  version = "3.12.17";
+
+  src = fetchurl {
+    url = "https://download.ankama.com/launcher/full/linux";
+    sha256 = "693cd3411c08a4ff3bae7fc627de7ccbd9ca6e4b99d706a2d9ae283412c9edb5";
```

```suggestion
    hash = "";
```
Run the build again with an empty hash, and copy the resulting `sha256-*` string.